assets/src/application/ui/input/EspressoButton/Tooltip.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import classNames from 'classnames';
+import { isMobile } from 'react-device-detect';
```

Comment:
hmmmm... if we have to go to this extent to do this then maybe we shouldn't bother...

I assumed things could be hidden solely using CSS but hadn't considered nesting and such...

That said, we should also consider what Gutenberg has since that will most likely already be available:
https://github.com/WordPress/gutenberg/tree/master/packages/viewport

and as a final thought, WordPress (at least in the admin) adds a `mobile` class to the `body` tag for small screens (although i don't know what pixel value that starts at) and it's super easy to detect that without having to load an entire library to do something that has essentially already been done:

```js
const body = document.getElementsByTagName("BODY")[0];
const isMobile = body  ? body.classList.contains('mobile') : false;
```

that would make a nice little utility, or could even be dumped into our config data

Reply:
thx, have used `@wordpress/viewport`

Reply:
I don't want to rely on a class on body element, the reason being that:
1. WP can update that name and break our functionality
2. it's quite easy to alter those classnames, with jQuery or native JS

Reply:
`@wordpress/viewport` gives us more control and we are not affected by outside changes of classnames and such

Reply:
great points, i agree

Reply:
ended up cloning `@wordpress/viewport` HOC, because of an error related to jest